config/routes.rb

```diff
@@ -51,6 +51,7 @@
   delete '/users/:user_id/remove-user', to: 'users#remove_user', as: :remove_user_post
   get '/users/:user_id/reset-user-password', to: 'users#show_reset_user_password', as: :reset_user_password
   post '/users/:user_id/reset-user-password', to: 'users#reset_user_password', as: :reset_user_password_post
+  get '/users/emails_csv', to: 'users#emails_csv', as: :emails_csv
```

Comment:
```suggestion
  get '/users/emails-csv', to: 'users#emails_csv', as: :emails_csv
```
I think the route should use hyphens instead of underscores.